root/static/scripts/edit/MB/edit.js

```diff
@@ -9,7 +9,7 @@
 import ko from 'knockout';
 import _ from 'lodash';
 
-import {hex_sha1} from '../../../lib/sha1/sha1';
+import {hexSha1} from '../../../lib/sha1/sha1';
```

Comment:
You can't import using a different name quite like that, since it's still `hex_sha1` in the file, but you can specify an alias:
```suggestion
import {hex_sha1 as hexSha1} from '../../../lib/sha1/sha1';
```

Reply:
Apologies, that was my bad.